<?xml version="1.0"?>
<launch>
  <node pkg="pcl_tutorial" type="voxel_filter_node" name="voxel_filter" output="screen">
    <remap from="point_cloud_in" to="/camera/depth_registered/points"/>
    <remap from="point_cloud_out" to="/voxel_filter/point_cloud_out"/>
    <param name="leafsize" type="double" value="0.01" />
  </node>
</launch>
